modules/selenium/src/test/java/org/testcontainers/containers/DefaultRecordingFileFactoryTest.java

```diff
@@ -0,0 +1,56 @@
+package org.testcontainers.containers;
+
+import lombok.Value;
+import org.junit.Test;
+import org.junit.runner.Description;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Date;
+
+import static java.lang.Boolean.FALSE;
+import static java.lang.Boolean.TRUE;
+import static java.lang.String.format;
+import static org.junit.Assert.assertEquals;
+import static org.junit.runner.Description.createTestDescription;
+
+@RunWith(Parameterized.class)
+@Value
+public class DefaultRecordingFileFactoryTest {
+
+    private static final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat("YYYYMMdd-HHmmss");
+
+    private final DefaultRecordingFileFactory factory = new DefaultRecordingFileFactory();
+    private final String methodName;
+    private final String prefix;
+    private final boolean success;
+
+    @Parameterized.Parameters
+    public static Collection<Object[]> data() {
+        Collection<Object[]> args = new ArrayList<>();
+        args.add(new Object[]{"testMethod1", "FAILED", FALSE});
+        args.add(new Object[]{"testMethod2", "PASSED", TRUE});
+        return args;
+    }
+
+    @Test
+    public void recordingFileThatShouldDescribeTheTestResultAtThePresentTime() throws Exception {
+        File vncRecordingDirectory = Files.createTempDirectory("recording").toFile();
+        Description description = createTestDescription(getClass().getCanonicalName(), methodName, Test.class);
+
+        File recordingFile = factory.recordingFileForTest(vncRecordingDirectory, description, success);
+
+        File expectedFile = new File(vncRecordingDirectory, format("%s-%s-%s-%s.flv",
+                prefix,
+                getClass().getSimpleName(),
+                methodName,
+                DATE_FORMAT.format(new Date()))
```

Comment:
I'm sorry to be pedantic, but I think there is a small risk that this test will be flaky... If it's running right on the transition from one second to the next, we're going to see a different date 😬 

Would it be possible to do some assertions on the file name instead, e.g. a string comparison that ignores the time part?

Reply:
Sure. Freezing the date during the test would be ideal.

Reply:
Actually, how about I check for both now and one second ago?